File: C++ for Everyone, 2nd Ed, Horstmann/Chapter2/P2-4.cpp
//Write a program that prompts the user for two integers and then prints
//• The sum
//• The difference
//• The product
//• The average

#include<iostream>
#include<cmath>
using namespace std;

int main()
{
    int number1;
    int number2;

    cout << "Enter number 1 and number 2\n";
    cin >> number1 >> number2;

    cout << "Type:\n 1 => sum, \n 2 => difference,\n 3 => product \n 4=> average\n";
    int user_input;
    cin >> user_input;

    double result = 0;
    switch(user_input)
        {
        case 1:
            result = number1 + number2;
            break;

        case 2:
            result = number1 - number2;
            break;

        case 3:
            result = number1 * number2;
            break;

        case 4:
            result = (number1 + number2) / 2;
            break;
        default:
            cout << "Invalid input\n";
        }

    cout << result;


    return 0;
}

File: Python For Everyone, Horstmann/Chapter2/P2.4.py
# Write a program that prompts the user for two integers and then prints
# • The sum
# • The difference
# • The product
# • The average
# • The distance (absolute value of the difference)
# • The maximum (the larger of the two)
# • The minimum (the smaller of the two)
# Hint: Python defines  max and  min functions that accept a sequence of values, each
# separated with a comma.

number1 = int(input("Enter number one: "))
number2 = int(input("Enter number two: "))

numbersSum = number1 + number2
numbersDifference = number1 - number2
numbersProduct = number1 * number1
numbersDistance = abs(number1 - number2)
numbersMax = max(number1, number2)
numbersMin = min(number1, number2)


print("The sum is: %d" % numbersSum)
print("The difference is: %d" % numbersDifference)
print("The product is: %d " % numbersProduct)
print("The distance is: %d" % numbersDistance)
print("The maximum is: %d" % numbersMax)
print("The minimum is: %d" % numbersMin)